docs/design/2021-03-09-security-enhanced-mode.md

```diff
@@ -99,18 +99,20 @@ The following tables will be hidden:
 * metrics_tables
 * tidb_hot_regions
 
-The following tables will be modified to hide columns:
+The following tables will be modified to hide columns unless the user has the `RESTRICTED_TABLES_ADMIN` privilege:
 
 * tikv_store_status
     * The address, capacity, available, start_ts and uptime columns will return NULL.
 * Tidb_servers_info
     * The “IP” column will return NULL.
 * cluster_* tables
     * The “instance” column will show the server ID instead of the server IP address.
+* statements_summary*
+    * The query_sample_text will return empty unless the user has the `PROCESS` privilege.
```

Comment:
I think this change may break compatibility.
If a customer does have access to the statements_summary table but does not have `PROCESS` privilege, they can not read the query_sample_text.

We may discuss whether there is another way to prevent `cloudAdmin` from accessing the original SQL text.
cc @breeswish

Reply:
Maybe column privilege is needed and we grant `cloudAdmin` access of all other columns except query_sample_text.

Reply:
I think it might be best to revert this part of the proposal if there is more discussion needed. We can handle in a followup PR instead.